pkg/operator/apis/monitoring/v1/types.go

```diff
@@ -1016,6 +1038,10 @@ type ClusterPodMonitoringSpec struct {
 	TargetLabels TargetLabels `json:"targetLabels,omitempty"`
 	// Limits to apply at scrape time.
 	Limits *ScrapeLimits `json:"limits,omitempty"`
+	// FilterRunning will drop any pods that are in the "Failed" or "Succeeded"
```

Comment:
Can we add to the comment what's the benefit for enabling this for the user?

Reply:
Done.